src/api/notification/routes/custom-notification.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    routes: [
+      {
+        method: 'GET',
+        path: '/notifications',
+        handler: 'notification.find',
```

Comment:
The frontend will use this endpoint to list notifications but filter by user ID?